Floofbot/Floofbot.csproj

```diff
@@ -5,20 +5,24 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+  </PropertyGroup>
+
```

Comment:
same comment as before, although not against updating but think it should be in a future PR

Reply:
I see. I'll set those back to their previous values.